io.catenax.manufacturing_capability/3.0.0/ManufacturingCapability.ttl

```diff
@@ -13,6 +13,7 @@
 # SPDX-License-Identifier: CC-BY-4.0
 #######################################################################
 
+
 @prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#>.

```

Comment:
Why are you making changes in 3.0.0 version when you are releasing 3.1.0 version?

Reply:
With commit d11fae4ac9db536175e4a163f859d3e7f276162d there should not be any more changes in 3.0.0.